pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -144,6 +144,9 @@ self: super: builtins.intersectAttrs super {
   gtk = disableHardening (addPkgconfigDepend (addBuildTool super.gtk self.gtk2hs-buildtools) pkgs.gtk2) ["fortify"];
   gtksourceview2 = addPkgconfigDepend super.gtksourceview2 pkgs.gtk2;
   gtk-traymanager = addPkgconfigDepend super.gtk-traymanager pkgs.gtk3;
+  gi-gdkx11 = addPkgconfigDepend super.gi-gdkx11 pkgs.gtk3;
```

Comment:
I believe that this override is no longer necessary due to https://github.com/NixOS/cabal2nix/commit/2923434f8d94be6943c8a7ecf2f7a1df5f3dcdc7. `gi-gdkx11` compiles fine in `haskell-updates` without this override.